zds/tutorialv2/views/statistics.py

```diff
@@ -85,23 +85,30 @@ def get_all_statistics(self, urls, start, end, methods):
             response_matomo = requests.post(url=self.matomo_api_url, data=data_request)
             data = response_matomo.json()
             if isinstance(data, dict) and data.get("result", "") == "error":
-                data = {}
                 self.logger.error(data.get("message", "Something failed with Matomo reporting system."))
                 messages.error(
-                    self.request, data.get("message", _(f"Impossible de récupérer les statistiques du site."))
+                    self.request, data.get("message", _("Impossible de récupérer les statistiques du site."))
```

Comment:
Je serais plutôt partisan d'avoir ici un message pour les humains systématiquement. C'est assez peu accueillant de se payer le message technique en anglais.

```suggestion
                    self.request, _("Impossible de récupérer les statistiques du site.")
```